docker_for_ctp/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM centos:7
+
+RUN yum install -y openssh-server openssh-clients vim
+
+RUN     ssh-keygen -q -b 4096 -N '' -t rsa -f /etc/ssh/ssh_host_rsa_key && \
+        ssh-keygen -q -b 1024 -N '' -t dsa -f /etc/ssh/ssh_host_dsa_key && \
+        ssh-keygen -q -b 521 -N '' -t ecdsa -f /etc/ssh/ssh_host_ecdsa_key
+
+RUN echo /usr/sbin/sshd >> /root/.bashrc && source /root/.bashrc
+
+RUN     sed -i \
+        -e 's~^#PermitRootLogin yes~PermitRootLogin yes~g' \
+        /etc/ssh/sshd_config
+
+CMD service sshd restart
+EXPOSE 22
+
+# Need package for test
+RUN yum install -y net-tools sudo gdb
+RUN yum install -y git wget tar java-1.6.0-openjdk.x86_64 java-1.6.0-openjdk-devel.x86_64 dos2unix
+RUN yum install -y gcc-4.4.7 gcc-c++ libtool-2.2.6 cmake make
```

Comment:
Are gcc-4.4.7 and gcc-c++ for running CTP? CUBRID Engine uses devtoolset-8-gcc and devtoolset-8-g++. Those versions are GCC 8.3.1

Reply:
The gcc-4.4.7 is the default version CentOS 7 as far as I know.
and It is used to CCI driver test.

If I version up to gcc then I can't guarantee to compatible the CCI drivier test cases.

Reply:
I see :-)